src/shared/ListRepo/ReposTable/ReposTable.spec.tsx

```diff
@@ -228,8 +252,71 @@ describe('ReposTable', () => {
       })
     })
 
+    describe('sorting table headers', () => {
+      const user = userEvent.setup()
+      it('sorts by name', async () => {
+        render(<ReposTable searchValue="" owner="owner1" />, {
+          wrapper: wrapper(repoDisplayOptions.ACTIVE.text),
+        })
+
+        const header = await screen.findByText(/Name/)
+        expect(header).toBeInTheDocument()
+        await user.click(header)
+        let tableElements = await screen.findAllByText(/Repo name/)
+        expect(tableElements[0]).toHaveTextContent('Repo name 1')
+        expect(tableElements[1]).toHaveTextContent('Repo name 2')
+        expect(tableElements[2]).toHaveTextContent('Repo name 3')
+
+        await user.click(header)
+        tableElements = await screen.findAllByText(/Repo name/)
+        expect(tableElements[0]).toHaveTextContent('Repo name 3')
+        expect(tableElements[1]).toHaveTextContent('Repo name 2')
+        expect(tableElements[2]).toHaveTextContent('Repo name 1')
```

Comment:
`m`: I would recommend not checking the table sort order and rather checking that the correct variables are passed. Since the sorting is being done on the API side these tests are really only checking that your mocked out response is sending the correct data. What I would suggest switching to is create a mock in the `setup` function and call it with the `req.variables` (i think that's right) inside the mocked GQL request and check that the the mock function was called with the right variables.

Same goes for the other sorting tests.

> This may be going against something I have said previously (at some point in time), but I think this is a better approach.